### DISCUSSION TITLE
auto refresh with dockerized dev server

### DISCUSSION THREAD

**Discussion by laurie71** [freshframework/fresh#917] [Q&A]
If I launch my Fresh project from the command line locally (with, for example, `deno run -A --watch="static/,routes/" --unstable dev.ts`) I get auto refresh in the browser; however, if I launch it inside a docker container using the same command, I don't. 

Does anyone have any ideas why that would be, or how to fix it?

**Comment by arisris** [comment]
> If I launch my Fresh project from the command line locally (with, for example, `deno run -A --watch="static/,routes/" --unstable dev.ts`) I get auto refresh in the browser; however, if I launch it inside a docker container using the same command, I don't.
> 
> Does anyone have any ideas why that would be, or how to fix it?

In production apps it should be set environtment variables DENO_DEPLOYMENT_ID with some random values like UUID? to disable live server. also you just need to `deno run main.ts` instead dev.ts

**Comment by laurie71** [comment]
I appreciate the response, but I'm not talking about production, I'm
talking about development with a containerized environment -- using
docker-compose to start my fresh app, a redis instance, a postgres
instance, etc. But when I do that, the containerized fresh app, launched in
its container using the same command line as I'd use to launch it locally,
doesn't provide hot reloading.

(Note that the app is running from a volume mounted from the host, so the
container hosted fresh app does see file changes, as evidenced by those
changes being reflected when I refresh the browser.)

I'm just wanting to enable hot reloading when running a development
environment that's set up this way.
